dramatiq/actor.py

```diff
@@ -133,6 +133,14 @@ def send_with_options(self, *, args=None, kwargs=None, delay=None, **options):
         Returns:
           Message: The enqueued message.
         """
+        for name in ["on_failure", "on_success"]:
```

Comment:
It might make sense to only run this block if the `Callbacks` middleware is set on `self.broker`.